mix.exs

```diff
@@ -61,6 +66,11 @@ defmodule Radiator.MixProject do
       {:poison, "~> 3.0"},
       {:timex, "~> 3.5"},
       {:cors_plug, "~> 2.0"},
+      # authentication
+      {:guardian, "~> 1.2"},
+      {:argon2_elixir, "~> 2.0"},
```

Comment:
`argon2_elixir` [requires](https://github.com/riverrun/comeonin/wiki/Requirements) gcc, make and glibc to build.

We could either:
1. ...make the C compiler a requirement for radiator-spark and define it as such in the Readme. Comeonin has this paragraph in its Readme:
  > Make sure you have a C compiler installed. See the [Comeonin wiki](https://github.com/riverrun/comeonin/wiki/Requirements) for details.
2. ...choose an other password hashing algorithm which is implemented in Elixir natively and does not rely on gcc.

While installing C on *nix systems is not that complicated, it is a larger act on Windows systems.

Reply:
I would go for 1) - as we probably won't be able to just rely on native elixir libraries anyways. E.g. as soon as we do any kind of audio processing I think we need to rely on c code. @eteubert what do you think?

Reply:
A valid concern but I also think that for our project scope we will likely need more libraries that depend on a C compiler later.